contrib/ansible/group_vars/all.yml

```diff
@@ -6,7 +6,7 @@ cluster_name: cluster.local
 # Account name of remote user. Ansible will use this user account to ssh into
 # the managed machines. The user must be able to use sudo without asking
 # for password unless ansible_sudo_pass is set
-#ansible_ssh_user: root
+ansible_ssh_user: fedora
```

Comment:
@jeffbean alert :) ! looks like we *cant* override all.yaml !? i didnt realize that... i guess all.yaml vals have precedence?  instead maybe these should go in some kind of inventory?  not an ansible star so might be missing something.